examples/joins/parameters/file1.txt

```diff
@@ -0,0 +1,5 @@
+1
```

Comment:
you're uploading the dummy files and having them create them with the makefile? Shouldn't we only do one or the other? Or were these pushed by accident?

Reply:
by accident 🤦‍♀